### PR TITLE
osx-ql 0.1 (new formula)

### PR DESCRIPTION
**Pull request by danielbayley**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/osx-ql.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class OsxQl < Formula
+  desc "Opens files with the Quick Look feature"
+  homepage "https://github.com/morgant/tools-osx#ql"
+  url "https://github.com/morgant/tools-osx.git",
+      tag:      "ql-0.1",
+      revision: "0a2ec702c03670530c0eb799049c81daf648f414"
```
Comment (chenrui333) [inline Formula/osx-ql.rb:6]: we'd prefer tarball over git checkout

```suggestion
  url "https://github.com/morgant/tools-osx/archive/refs/tags/ql-0.1.tar.gz"
  sha256 "16c079be28651879325d7bec0863d61341752c2e6e1373e18cedc8e72d7c75eb"
```

**Comment by danielbayley** [comment]
**Note** that [this](https://github.com/morgant/tools-osx/blob/master/src/ql#LL17C6-L17C11) `if` condition [fails](https://github.com/Homebrew/homebrew-core/actions/runs/4151948450/jobs/7182609593#step:6:75) the CI tests on Ventura (^`13`).

**Comment by danielbayley** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

Will fix…

**Comment by danielbayley** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

Will fix…

**Pull request closed without merging**